### ISSUE TITLE
Web crawler is not hitting my og meta tags

### ISSUE THREAD

**Issue by zdenham** [lookback/meteor-seo#16]
Using meteor-seo to attempt to get dynamic facebook dialog boxes. Also using spiderable. When I load the page I see all the beautiful meta tags with og:image title etc... but it doesn't appear that the Facebook spider gets them because when I use the open graph debugger they are not picked up. 

I'm assuming that the spider is hitting my site before the tags are rendered, how do I fix this?

**Comment by johanbrook** [comment]
Have you tried appended `?_escaped_fragment_=` to the URL you're checking in the FB debugger?

**Comment by zdenham** [comment]
Yes, I had tried that, and made some progress by switching from spiderable to ongoworks: spiderable. But still having the problem that facebook is picking up localhost for og: url. Very odd because it isn't hardcoded anywhere.

Here is the site: url: https://beta.visualify.xyz . Let me know if anything seems obvious that I didn't catch.

**Comment by johanbrook** [comment]
Not sure if I can sole this specific problem, but I know this package has trouble running on PhantomJS 1.9.x. It works better on 2.0.
